src/cdk/drag-drop/drag-drop.md

```diff
@@ -238,3 +238,10 @@ whenever an item is about to be moved into a new index. If the predicate returns
 item will be moved into the new index, otherwise it will keep its current position.
 
 <!-- example(cdk-drag-drop-sort-predicate) -->
+
+### Reordering table rows
+Adding `cdkDropList` on the `mat-table` element and the `cdkDrag` on the `mat-row` element groups the rows into a
+reorderable collection. Rows will automatically rearrange as an element moves. You need to
+listen to the `cdkDropListDropped` event to update the data model once the user finishes dragging.

```

Comment:
This describes how the drag&drop works in general. I think we can just say something about how drag&drop can be integrated into a table.

Reply:
How does this sound?

"Angular Material provides seamless integration of drag-and-drop functionality into tables, by adding the `cdkDropList` directive to your mat-table and handling the `(cdkDropListDropped)` event, you can enable drag-and-drop interactions within your table. This allows users to reorder rows or perform other custom actions with ease."

Reply:
Sounds good